content/for-administrators/how-to-guides/certificate-management/tls-certs.md

```diff
@@ -0,0 +1,104 @@
+# Configuring TLS certificates using Infra GitOps
+
+This document will explain a step-by-step approach of configuring TLS certificates for different tenants using Infra GitOps.
+
+1. First step is to navigate to correct path inside your Infra GitOps repository. For this example we are using following path:
+`<environment>/tenant-operator-config/templates/`
+Here `<environment>` correspond to the cluster where you want to deploy this.
+1. In this directory we need to create 2 resources [`Template`](https://docs.stakater.com/mto/main/crds-api-reference/template.html) and [`TemplateGroupInstance`](https://docs.stakater.com/mto/main/crds-api-reference/template-group-instance.html). A detailed explanation about why each resource is needed and what are the contents of each resource is given below:
+
+- ## Template
+
+    This resource is reponsible for keeping a record (template) or underlying resources (YAML files) that needs to be deployed to tenant namespaces.
```

Comment:
just wonder why you indent this text?

Reply:
The reason for indenting this text was to cover this text in current bullet point under headings